test/manual/amp-video-multiple-selectors.amp.html

```diff
@@ -0,0 +1,341 @@
+<!--
+  This is a dynamic file. It can be run statically, or can be generated for
+  multiple extensions through the server task.
+-->
```

Comment:
This comment is true for the file that this was copied from, but not for this one 🙂
```suggestion
```